web/package.json

```diff
@@ -249,7 +249,9 @@
     "xml2js": "0.5.0",
     "jsonwebtoken": "9.0.0",
     "moment": "2.29.4",
-    "qs": "6.5.3"
+    "qs": "6.5.3",
+    "@babel/helpers": "^7.26.10",
+    "@babel/runtime": "^7.26.10"
```

Comment:
why do we need them here?

Reply:
also these two libs should be placed in a correct order, otherwise any other `yarn install` will reorder them producing unnecessary diff in unrelated PR

Reply:
They are needed because they are transitive dependencies so, if not forced, the vulnerability is not fixed given any package using a lower version will still be using it